use_exact_time: false
buffer_size: 200
image_width: 640
image_height: 480